README.rst

```diff
@@ -803,7 +803,7 @@ data from Zyte API in page objects. Enable it in the Scrapy settings::
         ZyteApiProvider: 500,
```

Comment:
I think in almost all cases the desired behavior is to prioritize ItemProvider; I can't think of a case where prioritizing ZyteApiProvider is what users need in practice. 

Should we set the priority in the example to the value which gets this behavior?